src/model.rs

```diff
@@ -84,5 +83,5 @@ impl NewJob {
 pub fn db_connection() -> SqliteConnection {
     let database_url = env::var("DATABASE_URL").expect("DATABASE_URL environment variable missing");
     SqliteConnection::establish(&database_url)
-        .expect(&format!("Unable to find DB: {}", database_url))
+        .unwrap_or_else(|_| panic!("Unable to find DB: {}", database_url))
```

Comment:
The hell is this?

Reply:
Well, ask clippy.

Reply:
But from what I can see it's to avoid evaluation of `format!`.